integration-tests/bats/keyless.bats

```diff
@@ -843,14 +843,6 @@ SQL
     [[ "${lines[1]}" = "1" ]] || false
     [[ "${lines[2]}" = "1" ]] || false
     [ "${#lines[@]}" -eq 3 ]
-
-    run dolt sql -q "describe select c0 from keyless where c1 = 1" -r csv
```

Comment:
Why removed test?

Reply:
Same for below

Reply:
bats is a bad place for query query plans that break often